snmplib/snmp_api.c

```diff
@@ -7146,7 +7152,7 @@ netsnmp_oid_find_prefix(const oid * in_name1, size_t len1,
     min_size = SNMP_MIN(len1, len2);
     for(i = 0; i < (int)min_size; i++) {
         if (in_name1[i] != in_name2[i])
-            return i;    /* '�' is the first differing subidentifier
```

Comment:
It is not clear to me what has been changed in this line?

Reply:
apparently the encoding of the character 'í', I can see if I can create a pull request/ change the commit to not include this change bit odd tho.. (Though I would believe the 'í' should be i actually, would that change be ok?)

Reply:
Thanks for the clarification. I think that change counts as a fix :-)